<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.jayhenri.cloud9.customer.controller.AddressControllerTest" tests="8" skipped="0" failures="0" errors="0" timestamp="2021-08-28T20:38:11" hostname="kde-laptop" time="0.043">
  <properties/>
  <testcase name="updateEmployeeThrowsEmployeeNotFoundException()" classname="org.jayhenri.cloud9.customer.controller.AddressControllerTest" time="0.032"/>
  <testcase name="updateReturnsWrongType()" classname="org.jayhenri.cloud9.customer.controller.AddressControllerTest" time="0.001"/>
  <testcase name="updateStore()" classname="org.jayhenri.cloud9.customer.controller.AddressControllerTest" time="0.001"/>
  <testcase name="updateStoreThrowsStoreNotFoundException()" classname="org.jayhenri.cloud9.customer.controller.AddressControllerTest" time="0.001"/>
  <testcase name="updateCustomerThrowsCustomerNotFoundException()" classname="org.jayhenri.cloud9.customer.controller.AddressControllerTest" time="0.001"/>
  <testcase name="updateThrowsInvalidAddressException()" classname="org.jayhenri.cloud9.customer.controller.AddressControllerTest" time="0.001"/>
  <testcase name="updateCustomer()" classname="org.jayhenri.cloud9.customer.controller.AddressControllerTest" time="0.001"/>
  <testcase name="updateEmployee()" classname="org.jayhenri.cloud9.customer.controller.AddressControllerTest" time="0.002"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
